libraries/core_libs/consensus/src/vote_manager/vote_manager.cpp

```diff
@@ -291,12 +291,17 @@ void VoteManager::verifyVotes(uint64_t pbft_round, size_t sortition_threshold, u
     }
     bool vote_is_valid = true;
     auto dpos_votes_count = dpos_eligible_vote_count(v->getVoterAddr());
-    if (v->getStep() == 1 && v->getType() == propose_vote_type) {
-      // We need to handle propose_vote_type
-      if (!dpos_votes_count) vote_is_valid = false;
-      dpos_votes_count = 1;
+    if (!dpos_votes_count) {
+      // After deep syncing, node could receive votes but still behind, may don't have vote sender state in table
+      LOG(log_nf_) << "Node may behind and still in syncing. Cannot find vote sender state in table " << v;
+      vote_is_valid = false;
```

Comment:
you can use continue here insetad of vote_is_valid + remove if on line 300 so we get rif of one nested if